Makefile

```diff
@@ -6,7 +6,7 @@ prod: tms
 # Other options
 all:  tms tms_nosand gar_nosand garlite_nosand empty sandopt
 
-tms: nd_hall_with_lar_tms_sand.gdml \
+tms: nd_hall_with_lar_tms_sand_new_window.gdml \
```

Comment:
If the new window is supposed to be the new default geometry, it should just be added to _all_ the build options that do not explicitly require the old window. Not as a single new geometry. E.g. like this it would not be build for the "anti-fiducial" version of the default geometry.

Please, if the new window is the new default, just change how the previously existing targets were build. Maybe add an additional target to build with the old window.